### DISCUSSION TITLE
Account has been flagged for no reason and Github support won't respond to ticket even after 40 days

### DISCUSSION THREAD

**Discussion by P0VVeR** [community/community#116897] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Guys Its been 40 days since i created a ticket and still i haven't receive a response from github.
My work is totally blocked because of this issue I can't pull push anything
I get this error all the time "You are marked as spam, and therefore cannot authorize a third party application."
Account: https://github.com/iamPedram1
TicketID: #2620051
It's driving me crazy I would really appreciate any possible help

**Comment by mateusloubach** [comment]
Unfortunately there isn't a workaround this. The ONLY way for this tag to be removed is by Github themselves. And not only that, I think it's ONLY possible through the tickets, and you've done that.

All we can do is wait now..
Also, it's becoming a trend now. If you look in the General questions tab, there are A LOT of suspended or spam tagged accounts. This should've made them look into it, but I wouldn't count on it.

**Reply by mateusloubach** [reply]
Also, this has also been happening to me, and, for me to at least keep my job going, I had to create another account and approve that account into making changes to projects, and if that couldn't work, I'd have to create a new repo. It's really frustrating because we lose all the available tools we have in this account (especially PRO account tools, such as Copilot).

**Reply by martincout** [reply]
Same here, I don't even have the option to create a 2FA ticket even though I have 2FA enabled. It's really frustrating, I might create another account if this is not solved.

**Comment by Cobeine** [comment]
same here
